deploy/assisted-image-service-service.yaml

```diff
@@ -13,6 +13,7 @@ spec:
       port: 8080
       protocol: TCP
       targetPort: 8080
+      nodePort: 30001
```

Comment:
Is this manifest only used by Kind?

Reply:
Currently not as minikube is still used for e2e tests